src/sentry/integrations/jira/integration.py

```diff
@@ -511,6 +511,23 @@ def sync_assignee_outbound(self, external_issue, user, assign=True, **kwargs):
                 }
             )
 
+    def _get_done_statuses(self):
```

Comment:
@macqueen do the statuses change each time? For instance are there different JIRA issues in the data given in the webhook? Looks like this might result in a lot of api calls.

Reply:
fixed here: https://github.com/getsentry/sentry/pull/9121